pom.xml

```diff
@@ -65,19 +65,24 @@
     <forkCount>${concurrency}</forkCount> <!-- may use e.g. 2C for 2 × (number of cores) -->
     <trimStackTrace>false</trimStackTrace> <!-- SUREFIRE-1226 workaround -->
     <!-- Deprecated findbugs. -->
-    <findbugs.failOnError>true</findbugs.failOnError>
+    <findbugs.failOnError>false</findbugs.failOnError>
     <findbugs.threshold>Medium</findbugs.threshold>
     <findbugs.effort>default</findbugs.effort>
     <!-- Whether the build should fail if SpotBugs finds any error. -->
-    <!-- It is strongly encouraged to leave it as true. Use false with care only in transient situations. -->
+    <!-- See violations.maxViolations -->
     <spotbugs.failOnError>${findbugs.failOnError}</spotbugs.failOnError>
     <!-- Defines a SpotBugs threshold. Use "Low" to discover low-priority bugs.
          Hint: SpotBugs considers some real NPE risks in Jenkins as low-priority issues, it is recommended to enable it in plugins.
       -->
     <spotbugs.threshold>${findbugs.threshold}</spotbugs.threshold>
     <!-- Defines a SpotBugs effort. Use "Max" to maximize the scan depth -->
     <spotbugs.effort>${findbugs.effort}</spotbugs.effort>
-    
+    <!--
+      Maximum allowed number of violations found with static code analysis, or else fail the build. 
+      It is strongly encouraged to leave it as 0. Use false with care only in transient situations.
+    -->
```

Comment:
It causes incompatibilities for those who set up the 'failOnError' flag. We need to alter max violations if it is set, e.g. via profiles

Reply:
Corrected.
`mvn install -Dspotbugs.failOnError=false` allows 999999 violations.
`mvn install -Dspotbugs.failOnError=true` allows 0 violations.